oxd-server/src/main/java/org/gluu/oxd/server/service/Rp.java

```diff
@@ -178,9 +179,9 @@
     @JsonProperty(value = "software_statement")
     private String softwareStatement;
     @JsonProperty(value = "custom_attributes")
-    private Map<String, String> customAttributes;
-    @JsonProperty(value = "request_uris")
-    private List<String> requestUris;
+    private Map<String, String> customAttributes = new HashMap<String, String>();
+    @JsonProperty(value = "client_request_uris")
```

Comment:
We can't rename internal json keys because it would introduce incompatibility. Better leave it as is. Methods and fields can be renamed.

Reply:
Changes done